apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -35,6 +39,7 @@ const DepartureList = ({
   directionId,
   headsign,
   alerts,
+  hasService,
```

Comment:
not without more refactoring of the components (since the no service message & the departure times render in the same wrapping HTML) (this goes for above too)